src/app/ui/components/virtuoso-wrapper-sheet.tsx

```diff
@@ -0,0 +1,27 @@
+import { ReactNode, useState } from 'react';
+
+import { Stack } from 'leather-styles/jsx';
+
+import { useViewportMinWidth } from '@app/common/hooks/use-media-query';
+import { useOnResizeListener } from '@app/common/hooks/use-on-resize-listener';
+
+interface VirtuosoWrapperSheetProps {
+  children: ReactNode;
+}
+
+export function VirtuosoWrapperSheet({ children }: VirtuosoWrapperSheetProps) {
+  const [key, setKey] = useState(0);
+  const isAtLeastMd = useViewportMinWidth('md');
+  const height = isAtLeastMd ? '70vh' : '100vh';
+
+  const onResize = () => {
+    setKey(Date.now());
+  };
```

Comment:
```suggestion
  function onResize() {
    setKey(Date.now());
  }
```

nit, but can you also explain why we need to update the key on resize? This seems like a bit of a hack? 🤔 

Could `useId` be used here instead?

Reply:
I believe this is done for re-render virtual list on resize
I tested without this and worked ok, prob was done for removed logic work ok

so, removed this